components/Article.jsx

```diff
@@ -5,28 +5,42 @@ import { Link } from 'react-router-dom';
 
 
 
+
 const Article = ({article, setArticle}) => {
     const { articleId } = useParams();
+    const [vote, setVote] = useState(0)
     useEffect(() => {
         fetchArticlesByID(articleId).then(({article}) => {
           setArticle(article);
+          setVote(article.votes)
         });
       }, [articleId]);
 
       if (!article) {
         return <div>Loading...</div>;
     }
+     const handleVote = (voteType) => {
+        const updatedVotes = voteType === 'up' ? vote + 1 : vote - 1
+        setVote(updatedVotes)
+        setArticle({...article, votes: updatedVotes})
+     }
     return(
-        <>
+    <>
         <div className="card-body text-center">
               <h2>{article.title}</h2>
               <p>{article.author}</p>
               <p>Topic: {article.topic}</p>
+              <p>{article.body}</p>
               <img src={article.article_img_url} alt={article.title} className="img-fluid" />
               <p>Votes: {article.votes}</p>
+              <button type="button" className="btn btn-outline-primary" onClick={()=> handleVote('up')}>Upvote</button>
+              <button type="button" className="btn btn-outline-primary" onClick={()=> handleVote('down')}>Downvote</button>
+
         </div>
-        <Link to={`/articles/${articleId}/comments`}>Comments</Link>
-        </>
+        {article.comment_count === 0 ? (<p> No comments found </p>) : <Link to={`/articles/${articleId}/comments`} >
+        Comments
```

Comment:
When I click comments, the comments do show, however, this then removes the votes buttons. It looks like it is routing me to a different place where the comments are shown. Ideally, you should conditionally render the comments list on the same SingleArticle component so the user isn't navigated to a different place.